### DISCUSSION TITLE
The mouse, keyboard, stops in the antivirus window. And it's already running as a service.

### DISCUSSION THREAD

**Discussion by ytoaa** [LizardByte/community#152] [Sunshine] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

I don't know if this only happens in the vaccine program. But the screen is moving. It just stops the mouse keyboard.
And it's already running as a service.
It was run as an installation version, not a portable version.

### Host Operating System

None

### Operating System Version

windows 10

### Architecture

None

### Sunshine commit or version

nightly 23.01.29

### Package

None

### GPU Type

None

### GPU Model

intel uhd graphic( intel n6005)

### GPU Driver/Mesa Version

n/a

**Comment by ReenigneArcher** [comment]
Complete the details of the template. The issue is not helpful if no one can replicate it.

Also, do not open multiple issues like this. If you want an issue re-opened talk to us in our discord server.

**Comment by ytoaa** [comment]
> Complete the details of the template. The issue is not helpful is no one can replicate it.
> 
> Also, do not open multiple issues like this. If you want an issue re-opened talk to us in our discord server.

The problem has been closed and reopened.
This shows the same symptoms even if you run it with administrator privileges.

**Reply by NextDev65** [reply, marked as answer]
Hey @YTOAA I don't know if we had the same issue, but my problem was that the client mouse input was not moving the host cursor (Win10). I solved it by adding the sunshine.exe as a 'File' type exclusion on Windows Defender.

[a comment by an automated account is not shown]
